login.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="css/styles.css">
+    <title>Store Manager</title>
+</head>
+
+<body>
+    <!-- content -->
+    <div class="content">
+        <div class="login_card col-5">
+            <div class="content_login_card card ">
+                <div>
+                    <div class=class="login_logo">
+                        <img src="images/storelogo.png" />
+                        
+                    </div>
+
+                    <h1 class="content__title text-center">Login to Your Account</h1>
+                </div>
+                <form class="content__form">
+
+                    <div class="input-group content__input-group--padding-right">
+                        <label for="login_email" class="text-slate"> Username</label>
+                        <input class="input content__login-input" type="text" id="login_username" name="login_username" />
+                    </div>
+
+                    <div class="input-group content__input-group--padding-right">
+                        <label for="login_password" class="text-slate">Password</label>
+                        <input class="input content__login-input" type="password" id="login_password" name="login_password" />
+                    </div>
+
+                    <div content__login-submit>
+                        <a href="profile.html" class=" btn btn-login">
+                            Login
+
+                        </a>
+                    </div>
+                </form>
+                <br/>
+                <p class="copyright">
+                    &copy; 2018 Store Manager
+                </p>
+            </div>
+        </div>
+    </div>
+</body>
+
+</html>
```

Comment:
Your UI design is amazing. Its good practice to put an extra line at the end of your file

Reply:
Hi Charity,
The design has an awesome appeal and your narrations sound great.